package.json

```diff
@@ -10,7 +10,8 @@
     "lint:fix": "npm run lint -- --fix",
     "prettier": "prettier --list-different './**/*.{js,jsx}'",
     "prettier-clean": "prettier --write './**/*.{js,jsx}'",
-    "cypress": "cypress open"
+    "cypress:open": "cypress open",
+    "CI-test-prod": "start-server-and-test 'yarn start' http://localhost:3000 'yarn run cypress run --spec './cypress/integration/*.spec.js''"
```

Comment:
`yarn start`? 

Could you use `yarn dev` instead so you don't need to build the site first?

Reply:
That is purely going off memory that `next start` starts the production server and serves a built site, and dev starts the dev server

Reply:
There also seems to be some overlap with this PR here https://github.com/twilio-labs/paste-prototype-kit/pull/9/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R16

Reply:
For context: I changed it to <Code>yarn</Code> start after I talked to Rich, since he pointed out that we'd want to test out the production build not the development one. That's also why @cdelaomartinez and I thought our two scripts wouldn't be overlapping: one would be for dev, one would be for production. If that seems unnecessary, then we should probably just go with @lilianaw and @cdelaomartinez 's script, since it does use <Code>yarn dev</Code> and you wouldn't have to run <Code>yarn build</Code> first to use it. Whattaya think @SiTaggart ?

Reply:
I guess that's fine. I'm not 100% sure I'm convinced, but I don't feel particularly strongly about it.

I might just change the script name to `cypress:prod` if that's the case.

Reply:
I double checked and this scrip't shouldn't conflict with the development test script.

Reply:
@SiTaggart haha alrighty, then I will change the script name to cypress:prod and consider that done. Thanks for double checking about the conflict, @cdelaomartinez!